be/src/olap/push_handler.cpp

```diff
@@ -761,6 +915,126 @@ OLAPStatus LzoBinaryReader::_next_block() {
+    dummy_id.hi = 0;
+    dummy_id.lo = 0;
+    TPlanFragmentExecParams params;
+    params.fragment_instance_id = dummy_id;
+    params.query_id = dummy_id;
+    TExecPlanFragmentParams fragment_params;
+    fragment_params.params = params;
+    fragment_params.protocol_version = PaloInternalServiceVersion::V1;
+    TQueryOptions query_options;
+    TQueryGlobals query_globals;
+    _runtime_state.reset(new RuntimeState(fragment_params, query_options, query_globals,
+                                          ExecEnv::GetInstance()));
+    DescriptorTbl* desc_tbl = NULL;
+    Status status = DescriptorTbl::create(_runtime_state->obj_pool(), t_desc_tbl, &desc_tbl);
+    if (UNLIKELY(!status.ok())) {
+        LOG(WARNING) << "Failed to create descriptor table, msg: " << status.get_error_msg();
+        return OLAP_ERR_PUSH_INIT_ERROR;
+    }
+    _runtime_state->set_desc_tbl(desc_tbl);
+    status = _runtime_state->init_mem_trackers(dummy_id);
+    if (UNLIKELY(!status.ok())) {
+        LOG(WARNING) << "Failed to init mem trackers, msg: " << status.get_error_msg();
+        return OLAP_ERR_PUSH_INIT_ERROR;
+    }
+    _runtime_profile.reset(new RuntimeProfile(_runtime_state->obj_pool(), "PushBrokerReader"));
+    _mem_tracker.reset(new MemTracker(-1));
+    _mem_pool.reset(new MemPool(_mem_tracker.get()));
+    _counter.reset(new ScannerCounter());
+
+    // init scanner
+    BaseScanner *scanner = nullptr;
+    switch (t_scan_range.ranges[0].format_type) {
+    case TFileFormatType::FORMAT_PARQUET:
+        scanner = new ParquetScanner(_runtime_state.get(),
+                                  _runtime_profile.get(),
+                                  t_scan_range.params,
+                                  t_scan_range.ranges,
+                                  t_scan_range.broker_addresses,
+                                  _counter.get());
+        break;
+    default:
+        LOG(WARNING) << "Unsupported file format type: " << t_scan_range.ranges[0].format_type;
+		return OLAP_ERR_PUSH_INIT_ERROR;
+	}
+    _scanner.reset(scanner); 
+    status = _scanner->open();
+    if (UNLIKELY(!status.ok())) {
+        LOG(WARNING) << "Failed to open scanner, msg: " << status.get_error_msg();
+        return OLAP_ERR_PUSH_INIT_ERROR;
+    }
+
+    // init tuple
+    auto tuple_id = t_scan_range.params.dest_tuple_id;
+    _tuple_desc = _runtime_state->desc_tbl().get_tuple_descriptor(tuple_id);
+    if (_tuple_desc == nullptr) {
+        std::stringstream ss;
+        LOG(WARNING) << "Failed to get tuple descriptor, tuple_id: " << tuple_id;
+        return OLAP_ERR_PUSH_INIT_ERROR;
+    }
+
+    int tuple_buffer_size = _tuple_desc->byte_size();
+    void* tuple_buffer = _mem_pool->allocate(tuple_buffer_size);
+    if (tuple_buffer == nullptr) {
+        LOG(WARNING) << "Allocate memory for tuple failed";
+        return OLAP_ERR_PUSH_INIT_ERROR;
+    }
+    _tuple = reinterpret_cast<Tuple*>(tuple_buffer);
+
+    _ready = true;
+	return OLAP_SUCCESS;
```

Comment:
You should do a code format by clang-format.

Reply:
if I use format this file, it will add many unrelated changes to this commit. May be I will format the code in another pr.